stockflux-securities-master/src/components/InputForm.js

```diff
@@ -10,19 +10,25 @@ import ToggleSwitch from "./ToggleSwitch";
 import Button from "./Button";
 import Confirmation from "./Confirmation";
 import { InputFormState } from "../enums";
+import { inputFormReducer } from "../reducers/inputFormReducer";
 
 const InputForm = ({ match }) => {
   const [name, setName] = useState("");
   const [exchange, setExchange] = useState("");
   const [symbol, setSymbol] = useState("");
   const [visible, setVisible] = useState(false);
   const [enabled, setEnabled] = useState(false);
-  const [state, setState] = useState(
-    match.params.securityId ? InputFormState.LOADING : null
-  );
-  const [messages, setMessages] = useState([]);
+
   const [redirect, setRedirect] = useState(false);
```

Comment:
I think this would be worth changing to a single state object in your big refactoring PR.